Vue InstantSearch/autocomplete-results-page/src/App.vue

```diff
@@ -15,38 +15,35 @@
         :search-client="searchClient"
         index-name="demo_ecommerce"
       >
-        <ais-configure
-          :hitsPerPage="5"
-          :restrictSearchableAttributes="['name']"
-        />
-        <ais-autocomplete>
-          <template slot-scope="{ currentRefinement, indices, refine }">
-            <vue-autosuggest
-              :suggestions="indicesToSuggestions(indices)"
-              @selected="onSelect"
-              :input-props="{
-                style: 'width: 100%',
-                onInputChange: refine,
-                placeholder: 'Search here…',
-              }"
-            >
-              <template slot-scope="{ suggestion }">
-                <ais-highlight
-                  :hit="suggestion.item"
-                  attribute="name"
-                  v-if="suggestion.item.name"
-                />
-                <strong>$ {{ suggestion.item.price }}</strong>
-                <img :src="suggestion.item.image" />
-              </template>
-            </vue-autosuggest>
-          </template>
-        </ais-autocomplete>
-      </ais-instant-search>
-      <ais-instant-search
-        :search-client="searchClient"
-        index-name="demo_ecommerce"
-      >
+        <ais-index index-name="demo_ecommerce">
```

Comment:
this method now always does two queries, of which one unchanged, since the autocomplete index doesn't use the parent query. Do we leave this, or revert to the old two-instance method?

Reply:
Could you help understand it a little?
![Kapture 2020-02-05 at 14 57 52](https://user-images.githubusercontent.com/499898/73848199-f6d06b80-4827-11ea-944b-fa6c374ebb10.gif)
When I tried, each stroke triggered only one query at a time. And when I removed the wrapping `ais-index`, it was the same and worked fine.

So my question is

1. What do you mean by two queries?
2. Why do we need `ais-index` component in this example?

Reply:
a single request, with two queries in it

Reply:
We don't _need_ ais-index, but it should be an example of not needing two search instances.